install/telemetries.csv

```diff
@@ -0,0 +1,2 @@
+Station,Location,Gain,Start Date,End Date
```

Comment:
same as in comment above, shall we add susource/sublocation? (not a big fan of that but thought was worth mentioning)

Reply:
My thinking about Gain was it being a physical setting (i.e. a switch), e.g. preamp.

This would imply flicking over to "Scale Factor".

Reply:
turns out my confusion was that I'd updated the docs for the preamp, but not the code (it didn't matter as the columns were the same).

I'll put in a PR with the name changes for preamp now.

Reply:
thanks!!